app/controllers/recurring_job/base_controller.rb

```diff
@@ -1,29 +0,0 @@
-module RecurringJob
```

Comment:
requiring an auth token is no longer limited to recurring jobs, so I moved this out and renamed it